cfn-resources/project/cmd/resource/resource.go

```diff
@@ -544,11 +562,11 @@ func getChangeInAPIKeys(groupID string, currentKeys []ProjectApiKey, oKeys []mon
 	return newKeys, changedKeys, removeKeys
 }
 
-func readTeams(teams []ProjectTeam) []*mongodbatlas.ProjectTeam {
-	var newTeams []*mongodbatlas.ProjectTeam
+func readTeams(teams []ProjectTeam) []admin.TeamRole {
+	var newTeams []admin.TeamRole
```

Comment:
```suggestion
	var newTeams []*admin.TeamRole
```

Reply:
knitpit PR title is used for changelog and 3rd person is suggested, e.g.: refactor: Migrates Project resource to new Atlas SDK

Reply:
Adjusted. Keep in mind that currently in CFN we don't have a changelog or release notes associated to each resource, we could create a ticket to investigate on this now that we mention the topic.

Reply:
ah, ok